webpack.config.js

```diff
@@ -0,0 +1,97 @@
+const path = require('path'),
+  webpack = require('webpack'),
+  CleanWebpackPlugin = require('clean-webpack-plugin'),
+  HtmlWebpackPlugin = require('html-webpack-plugin'),
+  ExtractTextPlugin = require('extract-text-webpack-plugin');
+
+const extractPlugin = new ExtractTextPlugin({filename: './assets/css/app.css'});
+
+const config = {
+
+  context: path.resolve(__dirname, 'src'),
+
+  entry: {
+    app: './index.js'
+  },
+
+  output: {
+    path: path.resolve(__dirname, 'dist'),
+    filename: './assets/js/[name].bundle.js'
+  },
+
+  module: {
+    rules: [
+
+      {
+        test: /\.js$/,
+        include: /src/,
+        exclude: /node_modules/,
+        use: {
+          loader: "babel-loader",
```

Comment:
```suggestion
          loader: 'babel-loader',
```

you everywhere use a single quote, so do the same here